tests/Mockery/Adapter/Phpunit/TestListenerTest.php

```diff
@@ -96,8 +96,24 @@ public function testMockeryIsAddedToBlacklist()
     {
         $suite = \Mockery::mock(\PHPUnit\Framework\TestSuite::class);
 
-        $this->assertArrayNotHasKey(\Mockery::class, \PHPUnit\Util\Blacklist::$blacklistedClassNames);
-        $this->listener->startTestSuite($suite);
-        $this->assertSame(1, \PHPUnit\Util\Blacklist::$blacklistedClassNames[\Mockery::class]);
+        if (method_exists(\PHPUnit\Util\Blacklist::class, 'addDirectory')) {
```

Comment:
Perhaps it would be better to split this up into two tests and skip one of them depending on PHPUnit version?

Reply:
I would agree :+1:

Reply:
@davedevelopment I can't find an easy way to check the minimum required version of PHPUnit is `9.1.3`, so maybe we can leave this as is?